src/player.js

```diff
@@ -1897,6 +2059,19 @@ export default class Player extends FakeEventTarget {
     };
   }
 
+  /**
+     }
+   * Checks auto play configuration and handles initialization accordingly.
+   * @returns {void}
+   * @private
+   */
+  _initAutoPlay(): void {
+    this._posterManager.show();
```

Comment:
why you changed the previous logic which shows the poster only for autoplay false?
maybe it could cause a flicker

Reply:
and add also isAudio to this condition, the same as we have  in the master today

Reply:
fixed